docker-compose/jaeger-docker-compose-es.yml

```diff
@@ -0,0 +1,99 @@
+---
+version: "3"
+
+services:
+  # Run elasticsearch first and wait a few seconds:
+  #   docker-compose up -d elasticsearch
+  # This step is required to satisfy health check.
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.3.1
+    networks:
+      - elastic-jaeger
+    ports:
+      - "127.0.0.1:9200:9200"
+      - "127.0.0.1:9300:9300"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    restart: on-failure
+    environment:
+      - cluster.name=jaeger-cluster
+      - discovery.type=single-node
+      - network.host=0.0.0.0
+      - transport.host=127.0.0.1
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - xpack.security.enabled=false
+    volumes:
+      - esdata:/usr/share/elasticsearch/data
+
+  jaeger-collector:
+    image: jaegertracing/jaeger-collector
+    ports:
+      - "14269:14269"
+      - "14268:14268"
+      - "14267:14267"
+      - "9411:9411"
+    networks:
+      - elastic-jaeger
+    restart: on-failure
+    environment:
+      - SPAN_STORAGE_TYPE=elasticsearch
+    command: [
+      "--es.server-urls=http://elasticsearch:9200",
+      "--es.num-shards=1",
+      "--es.num-replicas=0",
+      "--log-level=error"
+    ]
+    depends_on:
+      - elasticsearch
+
+  jaeger-agent:
+    image: jaegertracing/jaeger-agent
+    hostname: jaeger-agent
+    command: [
+      "--collector.host-port=jaeger-collector:14267",
+      "--discovery.conn-check-timeout=3s",
+      "--discovery.min-peers=0"
+    ]
+    ports:
+      - "5775:5775/udp"
+      - "6831:6831/udp"
+      - "6832:6832/udp"
+      - "5778:5778"
+    networks:
+      - elastic-jaeger
+    networks:
+      - elastic-jaeger
+    restart: on-failure
+    environment:
+      - SPAN_STORAGE_TYPE=elasticsearch
+    depends_on:
+      - jaeger-collector
+
+  jaeger-query:
+    image: jaegertracing/jaeger-query
+    environment:
+      - SPAN_STORAGE_TYPE=elasticsearch
+      - no_proxy=localhost
+    ports:
+      - "16686:16686"
+      - "16687:16687"
+    networks:
+      - elastic-jaeger
+    restart: on-failure
+    command: [
+      "--es.server-urls=http://elasticsearch:9200",
+      "--span-storage.type=elasticsearch",
+      "--log-level=error"
```

Comment:
I don't think it should be set, unless there's a good reason to default to `error` for the `docker-compose` but not in all other deployment options (like, [Kubernetes](https://github.com/jaegertracing/jaeger-kubernetes/blob/master/production-elasticsearch/configmap.yml#L32) or [OpenShift](https://github.com/jaegertracing/jaeger-openshift/blob/master/production/configmap-elasticsearch.yml#L30))